wormhole-connect/src/views/Bridge/GasOptions.tsx

```diff
@@ -165,26 +165,44 @@ function GasOptions(props: { disabled: boolean }) {
     description: '',
     options: [] as OptionConfig[],
   });
-  const selectedOption = useSelector(
-    (state: RootState) => state.transferInput.route,
-  );
-  const { token, fromNetwork, toNetwork, automaticRelayAvail, gasEst } =
+
+  const { token, fromNetwork, toNetwork, automaticRelayAvail, gasEst, route } =
     useSelector((state: RootState) => state.transferInput);
+  const automaticOrManual =
+    route === Route.BRIDGE
+      ? Route.BRIDGE
+      : route === Route.RELAY
+      ? Route.RELAY
+      : route === Route.CCTPManual
+      ? Route.BRIDGE
+      : Route.RELAY;
```

Comment:
Can we change this variable to `isAutomatic` as a boolean?  Could probably add this value on the route as well for easy access